discovery-provider/src/models/models.py

```diff
@@ -581,6 +581,17 @@ def __repr__(self):
 play_item_id={self.play_item_id},\
 count={self.count}>"
 
+class LatestSlots(Base):
```

Comment:
lets make this its own file and add a few comments explaining the intent behind this table?